src/lib/expansion/expansion-panel-header.html

```diff
@@ -3,5 +3,7 @@
   <ng-content select="mat-panel-description"></ng-content>
   <ng-content></ng-content>
 </span>
-<span [@indicatorRotate]="_getExpandedState()" *ngIf="_showToggle()"
-      class="mat-expansion-indicator"></span>
+<div class="mat-expansion-indicator-container" [style.order]="_indicatorOrder()">
```

Comment:
It might be more appropriate to toggle a class that sets the `order`. It also makes it easier to handle RTL.

Reply:
Good call